store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReader.java

```diff
@@ -114,6 +117,43 @@ public static CarbonReaderBuilder builder(String tablePath) {
     return builder(tablePath, tableName);
   }
 
+  /**
+   * Return a new list of {@link CarbonReader} objects
+   *
```

Comment:
Add a clear description, mention what happens if splits greater than the number of files and what happens if splits are lesser than the number of files

Reply:
Done!